### DISCUSSION TITLE
Suppress Protocol Window during import

### DISCUSSION THREAD

**Discussion by Marty56** [mytourbook/mytourbook#1210] [Q&A]
when importing a tour, a protocol window is generated providing no useful information (for me). 
For me this is superficial and useless.
Is there an option that suppresses this protocol window.

**Comment by FJBDev** [comment]
@Marty56 Can you show a screenshot of the "protocol window" you're talking about ?

**Comment by wolfgang-ch** [comment]
Have you disabled this checkbox ?

![image](https://github.com/mytourbook/mytourbook/assets/1283445/5da7e506-6883-4964-85bb-45302da933dd)

**Comment by Marty56** [comment]
Thanks for the answer.

Yes, I had it enabled. What I had to do.
1. enabled it and 
2. then disabled it again.

Now the protocol log is gone.

PS: I am on MacOS. Unsupported I know.

**Comment by Marty56** [comment]
Import is working without the protocol windows.
But the protocol window is opening again when deleting a track.
I did not find a setting to disable this behaviour

**Comment by Marty56** [comment]
Leider funktioniert das Ganze nicht auf MacOS
Jedesmal wenn ich MyTourBook neu öffne, wird dieses Protokoll Fenster wieder neu angezeigt.
In den Settings ist es disabled. 

Wenn ich dann die checkmark rein und wieder raus nehme, funktioniert es solange ich MyTourBook nicht schließe.

Außerdem kommt dieses Protokollfenster auch immer wenn ich einen Track lösche.

**Reply by wolfgang-ch** [reply]
I cannot verify that when the log is disabled for the tour import that it will be opened when MT is opened or a tour is imported. Can you create a video which shows this issue.

>Außerdem kommt dieses Protokollfenster auch immer wenn ich einen Track lösche.

There are at least 27 possibilities that the log view could be opened

![image](https://github.com/mytourbook/mytourbook/assets/1283445/984671d2-1d98-40ee-9ae4-df295c641113)

**Comment by wolfgang-ch** [comment]
Implemented in https://github.com/mytourbook/mytourbook/issues/1222
